views/leaguesearch.hbs

```diff
@@ -1,9 +1,20 @@
 <div class="league-middle">
   <div class="container-league">
     <div id="league-type"></div>    
-    <div class="column-left all-columns"><div class="league-align-center"><p class="all-icons roster-icon" id="rosters-league-page" data-toggle="popover" data-container="body" data-placement="left" title="Team roster quizzes" data-content="See if you can name all the players on a specific team" data-trigger="hover"></p></div>{{#each league}}  {{/each}}</div>
+    <div class="column-left all-columns"><div class="league-align-center"><p class="all-icons roster-icon" id="rosters-league-page" data-toggle="popover" data-container="body" data-placement="left" title="Team roster quizzes" data-content="See if you can name all the players on a specific team" data-trigger="hover"></p></div>
+    
+    <div class="top-list-contain">
+	    <ul id="top-teams-league-page">    
+	    {{#each popular_teams}}  
+	    	<li><a href="/quiz?[QUIZ ID GOES HERE]&amp;" + "team_id=" + {{team_id}} + "league=" + {{this.league}} + '"'>{{this.team_name}}</a></li>
```

Comment:
Don't make this an a tag. and drop the href. Add the class "create-quiz" and the data attribute ```data-id={{team_id}} ```. This will then hookup to the already in place javascript that will create quizzes for us based on team id